sauce/features/budget/enter-to-move/index.js

```diff
@@ -43,7 +39,19 @@ export class EnterToMove extends Feature {
       changedNodes.has(CATEGORY_DROPDOWN) &&
       changedNodes.has(BUTTON_PRIMARY)
     ) {
-      document.addEventListener('keydown', this.onKeyDownHandler, false);
+      // Ready to click button
+      document.addEventListener('keydown', this.onKeyDown, false);
+    }
+
+    if (
+      this.modalIsOpen &&
+      changedNodes.has(BUTTON_PRIMARY) &&
+      changedNodes.has(BUTTON_CANCEL) &&
+      changedNodes.has(MODAL_CONTENT)
+    ) {
```

Comment:
If there's a better way to check that the Move Money modal is closed, I'm open for suggestions!